src/Traits/MutatesMigrationCommands.php

```diff
@@ -14,10 +14,11 @@
 
 namespace Hyn\Tenancy\Traits;
 
-use Hyn\Tenancy\Contracts\Repositories\WebsiteRepository;
+use InvalidArgumentException;
 use Hyn\Tenancy\Database\Connection;
+use Illuminate\Contracts\Events\Dispatcher;
 use Illuminate\Database\Migrations\Migrator;
-use InvalidArgumentException;
+use Hyn\Tenancy\Contracts\Repositories\WebsiteRepository;
```

Comment:
In the future, please try not to commit superfluous changes like this.  Makes it very difficult to review.